mapie/utils.py

```diff
@@ -202,3 +202,28 @@ def check_n_features_in(
                 "X.shape and estimator.n_features_in_."
             )
     return n_features_in
+
+
+def check_calcul_of_quantile(alphas: np.ndarray, n: int) -> None:
+    """
+    Check if the quantile is calculable
```

Comment:
Je décrirais un peu plus ce que fait la fonction: "Check if the quantile can be computed based on the number of samples and the alpha value."

Reply:
corrigé